modules/bibauthorid/lib/bibauthorid_hoover.py

```diff
@@ -0,0 +1,721 @@
+##
+## This file is part of Invenio.
+## Copyright (C) 2014 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+import sys
+from time import time
+from invenio.search_engine import get_record, \
+    perform_request_search, search_unit_in_bibxxx
+
+from invenio.bibauthorid_logutils import Logger
+
+from invenio.dbquery import run_sql
+from invenio.bibauthorid_dbinterface import get_existing_authors, \
+    get_canonical_name_of_author, get_papers_of_author, \
+    get_all_paper_data_of_author, \
+    get_signatures_of_paper, get_author_info_of_confirmed_paper, \
+    _get_external_ids_from_papers_of_author, get_claimed_papers_of_author, \
+    get_inspire_id_of_signature, populate_partial_marc_caches, move_signature, \
+    add_external_id_to_author, get_free_author_id, get_inspire_id_of_author, \
+    get_orcid_id_of_author, destroy_partial_marc_caches, get_name_by_bibref
+from invenio.bibauthorid_general_utils import memoized
+import invenio.bibauthorid_dbinterface as db
+import invenio.bibauthorid_config as bconfig
+from invenio.bibauthorid_webapi import get_hepnames, add_cname_to_hepname_record
+from invenio.bibcatalog import BIBCATALOG_SYSTEM
+from invenio.bibauthorid_hoover_exceptions import *
+
+logger = Logger('Hoover')
+
+try:
+    from collections import defaultdict
+except ImportError:
+    from invenio.bibauthorid_general_utils import defaultdict
+
+
+def open_rt_ticket(e, debug_log=False, queue='Test'):
```

Comment:
@petros-ioannidis the queue is still not configurable. Can you make it configurable?